sync-core/src/main/java/com/cloudant/sync/sqlite/SQLDatabaseQueue.java

```diff
@@ -128,6 +135,11 @@ public void run() {
             }
         });
         queue.shutdown();
+        try {
+            queue.awaitTermination(200, TimeUnit.SECONDS);
```

Comment:
I guess we could call `shutdownNow()` in the task running on the queue,the tests pass with that behavior, at least on JavaSE.